support-frontend/assets/components/thankYou/thankYouModule.tsx

```diff
@@ -260,12 +270,23 @@ function ThankYouModule({
 	const maybePaddingRight =
 		!hasImagery && (isDownloadModules ? paddingRightApps : paddingRight);
 
+	const isNewspaperArchiveBenefit = moduleType === 'newspaperArchiveBenefit';
+	const resizeContainer = isNewspaperArchiveBenefit ? sizeContainer : css``;
```

Comment:
This bit of styling logic is quite confusing - would be great if we could simplify it a bit

Reply:
Hmm, was trying to keep the `resizeContainer` a separate css style so could apply only to `newspaperArchive` not `appDownload`? Can pass `isNewspaperArchiveBenefit` into the css call itself. James never liked that however saying was slower - happy to do that way if your sure?

Reply:
Let's keep it efficient 👍